package.json

```diff
@@ -125,12 +127,32 @@
                 }
             }
         },
+        "breakpoints": [
+            {
+                "language": "markdown"
+            },
+            {
+                "language": "typescript"
+            },
```

Comment:
Why markdown? Also pretty sure this was already resolved, but Typescript doesn't currently work.